internal/app/api/handler/index/index.go

```diff
@@ -30,3 +38,69 @@ func (web *Handler) Index(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func (web *Handler) AddFriend(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+	var friend *types.Friend
+
+	if err := json.NewDecoder(r.Body).Decode(&friend); err != nil {
+		respond.Error(w, err, http.StatusBadRequest)
+		return
+	}
+	friend.Id = bson.NewObjectId()
+	if err := web.repo.InsertFriend(friend); err != nil {
+		respond.Error(w, err, http.StatusInternalServerError)
+		return
+	}
+
+	respond.JSON(w, http.StatusCreated, map[string]string{"result": "success"})
```

Comment:
a POST should return id of the created object. It's useful for the UI to know the id to process next steps.

Reply:
Fixed